portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,36 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <title>My Portfolio | Peter Wu</title>
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
```

Comment:
You're not contributing to google3 codebase, so feel free to ignore my comment :)

To avoid page loading blocked by script execution, if your website page doesn't require async loading like what you have here, it's recommended to put this script tag at the bottom to speed up the initial page rendering.

Reply:
Done (this was already existing in the provided portfolio boilerplate), good point!